pages/common/sed.md

```diff
@@ -2,21 +2,21 @@
 
 > Run replacements based on regular expressions
 
-- replace all occurrences of a string in a file, and print the result
+- replace the first occurrence of a string in a file, and print the result
 
-`sed 's/{{find}}/{{replace}}/g' {{filename}}`
+`sed 's/{{find}}/{{replace}}/' {{filename}}`
 
-- replace all occurrences of a string in a file, and overwrite the file
+- replace all occurrences of a string in a file, overwriting the file
   contents
 
-`sed -i '' 's/{{find}}/{{replace}}/g' {{filename}}`
+`sed --in-place 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
That's a pretty good guideline to follow, I'll fully agree with that :). It's good to be able to track back the meaning of some option. 

> Replace all occurrences of a string in place

It is a tad awkward isn't it. I like how you managed to have the verbose option in the explanation and the short version in the actual command even though that makes the explanation a bit more verbose. If you agree, let's go with that one.